test/e2e/framework/fips.go

```diff
@@ -0,0 +1,89 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	fipsNamespace     = "kube-system"
+	fipsConfigMapName = "cluster-config-v1"
+)
+
+func FindFipsConfig(cluster ClusterIndex) bool {
+	configMap, err := KubeClients[cluster].CoreV1().ConfigMaps(fipsNamespace).Get(context.TODO(), fipsConfigMapName, metav1.GetOptions{})
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			return false
+		}
+	}
+
+	if strings.Contains(configMap.Data["install-config"], "fips: true") {
+		return true
+	}
+
+	return false
+}
+
+func (f *Framework) FindFipsEnabledCluster() int {
+	for idx := range TestContext.ClusterIDs {
+		fipsEnabled := FindFipsConfig(ClusterIndex(idx))
+		if fipsEnabled {
+			return idx
+		}
+	}
+
+	return -1
+}
+
+func (f *Framework) TestGatewayNodeFipsMode(cluster ClusterIndex, gwPod string) error {
+	ctx := context.TODO()
+	cmd := []string{"sh", "-c", "ipsec pluto --selftest"}
```

Comment:
Why not invoke `ipsec` directly?

Reply:
@skitt 

Could you advice what is the way to do it, please?

Reply:
I was thinking of

    cmd := []string{"ipsec", "pluto", "--selftest"}

Reply:
Ah, you mean to call directly the command.
I thought there is some kind of api.
I had an issue to do it.

Will recheck and report back.

Reply:
Changed to invoke ipsec directly.